test/tools/isoltest.cpp

```diff
@@ -146,6 +152,16 @@ class TestTool
 
 bool TestTool::m_exitRequested = false;
 
+void TestTool::printOptions(ostream& _stream, string const& _linePrefix, bool) const
+{
+	_stream << _linePrefix << "// ====" << endl;
+	_stream << _linePrefix << "// evmVersion: " << m_options.evmVersion().name() << endl;
+	_stream << _linePrefix << "// optimize: " << m_options.optimize << endl;
+	_stream << _linePrefix << "// useABIEncoderV1: " << m_options.useABIEncoderV1 << endl;
+	_stream << _linePrefix << "// batches: " << m_options.batches << endl;
+	_stream << _linePrefix << "// selectedBatch: " << m_options.selectedBatch << endl;
```

Comment:
This format makes it look like the expectations you can add in the test files but you cannot use it there. I guess we can do it like this too but I was really thinking about something more compact since these values are just extra information that you do not always need. They should not be some prominent.

Maybe it would be better to have them on a single line? I.e. something like:
```
Run settings: EVM=london, optimize, batch=3/125
```
with boolean values like `optimize` and `abiencoderv1` just being omitted when false.

Or, alternatively:
```
Run settings: evmVersion=london, optimize=true, abiencoderv1=false, batch=3/125
```

Reply:
I would prefer the second form because you leave no room for doubt about the omitted/false values...

Reply:
Sure, I'm fine with either.